pkg/api/api.go

```diff
@@ -326,7 +332,11 @@ func (a API) Invoke(w http.ResponseWriter, r *http.Request) {
 	}
 	evt := event.NewInvocationEvent(newInvOpts)
 
-	evtID, err := a.handler(r.Context(), &evt)
+	seed := event.SeededIDFromString(
+		r.Header.Get("x-inngest-idempotency-key"),
```

Comment:
(Discussed - using as a reminder)
```suggestion
		r.Header.Get("x-inngest-event-id-seed"),
```
Nit: Could this also be a const somewhere?